### ISSUE TITLE
Error with companion service

### ISSUE THREAD

**Issue by marculos** [alexa/alexa-avs-sample-app#407]
Hi, I am getting an error after logging in to Amazon. "There is most likely a problem with the alexa-voice-service-companion-service package". I have reinstalled NPM. All the paths in the config look fine. Any ideas ?
Maybe something to do with the certificates ?

/**
 * @module
 * This module defines the settings that need to be configured for a new
 * environment.
 * The clientId and clientSecret are provided when you create
 * a new security profile in Login with Amazon.  
 * 
 * You will also need to specify
 * the redirect url under allowed settings as the return url that LWA
 * will call back to with the authorization code.  The authresponse endpoint
 * is setup in app.js, and should not be changed.  
 * 
 * lwaRedirectHost and lwaApiHost are setup for login with Amazon, and you should
 * not need to modify those elements.
 */
var config = {
    clientId: "amzn1.application-oa2-client.5b59e03feada487e99d449db7e27ea95",
    clientSecret: "XXX",
    redirectUrl: 'https://localhost:3000/authresponse',
    lwaRedirectHost: "amazon.com",
    lwaApiHost: "api.amazon.com",
    validateCertChain: true,
    sslKey: "/home/pi/Desktop/alexa-avs-sample-app/samples/javaclient/certs/server/node.key",
    sslCert: "/home/pi/Desktop/alexa-avs-sample-app/samples/javaclient/certs/server/node.crt",
    sslCaCert: "/home/pi/Desktop/alexa-avs-sample-app/samples/javaclient/certs/ca/ca.crt",
    products: {
        "my_device": ["1xxxxxxxxxx"],
    },
};

module.exports = config;

.......................NPM_DEBUG

0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/nodejs', '/usr/bin/npm', 'start' ]
2 info using npm@1.4.21
3 info using node@v0.10.29
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info prestart alexa-voice-service-sample-companion-service@1.0.0
6 info start alexa-voice-service-sample-companion-service@1.0.0
7 verbose unsafe-perm in lifecycle true
8 info alexa-voice-service-sample-companion-service@1.0.0 Failed to exec start script
9 error alexa-voice-service-sample-companion-service@1.0.0 start: `node ./bin/www`
9 error Exit status 8
10 error Failed at the alexa-voice-service-sample-companion-service@1.0.0 start script.
10 error This is most likely a problem with the alexa-voice-service-sample-companion-service package,
10 error not with npm itself.
10 error Tell the author that this fails on your system:
10 error     node ./bin/www
10 error You can get their info via:
10 error     npm owner ls alexa-voice-service-sample-companion-service
10 error There is likely additional logging output above.
11 error System Linux 4.4.34+
12 error command "/usr/bin/nodejs" "/usr/bin/npm" "start"
13 error cwd /home/pi/Desktop/alexa-avs-sample-app/samples/companionService
14 error node -v v0.10.29
15 error npm -v 1.4.21
16 error code ELIFECYCLE
17 verbose exit [ 1, true ]

**Comment by PaulWebster** [comment]
I don;t know what the underlying issue is but you should not publish you clientSecret

**Comment by marculos** [comment]
thanks for the hint. have removed the secret

**Comment by phelps-oc** [comment]
someone mentioned in another post about extra commas. after "] and after the }. try removing those.
"my_device": ["1xxxxxxxxxx"],
},

**Comment by marculos** [comment]
Hi, I have tried removing the comma and get the same error :(

**Comment by phelps-oc** [comment]
Hi marculos, I got too frustrated with all these problems so I started over from scratch following these directions.

https://github.com/alexa/alexa-avs-sample-app/wiki/Raspberry-Pi#step-1-setting-up-your-pi

I formatted the sd card and started all over and everything worked perfectly. This was my first Pi project so I could easily start over. The problem is there are many different how to Alexa tutorials out there and they leave out steps or start off in a different order of operations and get noobs like me running down rabbit hole after rabbit hole trying to figure out why I get error after error.

Hope that helps! Good luck!
